src/properties/property-effects.html

```diff
@@ -983,25 +1217,6 @@
         }
       }
 
-      /**
-       * Adds to the default implementation in `PropertyAccessors` by clearing
-       * any locally cached path values if a root object has been set, as that
-       * invalidates any descendant paths of that object.
-       *
-       * @override
-       */
```

Comment:
Clear path no longer needed; replaced with temp `__dataTemp` cache clearing in `_flushProperties`